framework/set/resources/proxy/createMainTF.go

```diff
@@ -44,8 +45,9 @@ func CreateMainTF(t *testing.T, terraformOptions *terraform.Options, keyPath str
 	tfBlockBody := tfBlock.Body()
 
 	instances := []string{rke2Bastion}
-	logrus.Infof("Creating AWS resources ...")
-	file, err := aws.CreateAWSResources(file, newFile, tfBlockBody, rootBody, terraformConfig, terratest, instances)
+
+	awsConnection := providers.TunnelToProvider(defaults.Aws)
```

Comment:
It can be here or in the the `TunnelToProvider` function, but can you add back the logrus message? Those were added so that if you have `standaloneLogging: false`, you can track where you are in the creation of Rancher.

This would apply for the rest of the files as well.